x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/elastic/ElasticInferenceServiceRerankRequestManager.java

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
We're trying to transition away from the request manager pattern to avoid the extra class since all the classes are pretty similar.

Here's an example of how we implemented it for voyageai: https://github.com/elastic/elasticsearch/pull/124512

And the rerank usage: https://github.com/elastic/elasticsearch/pull/124512/files#diff-3493deea8c9fd5276917f1f8a9d7f008268c34b61f230885df0643de10f19fffR71

Reply:
[Remove ElasticInferenceServiceRerankRequestManager](https://github.com/elastic/elasticsearch/pull/126022/commits/2b89009b0c3a924b09a87ab5ca139a87f8eb3bbd)